addons/website_sale/static/src/js/location_selector/location_selector_dialog/location_selector_dialog.scss

```diff
@@ -0,0 +1,23 @@
+.o_locationSelector_number {
+    width: 2ch;
+    font-variant-numeric: tabular-nums;
+}
+
+.o_locationSelector_details {
+    transform: translateY(-100%);
+}
+
+.collapsed i.o_locationSelector_hours_caret {
+    transform: rotate(90deg) !important;
+}
+
+.nav-tabs .o_locationSelector_mobile_tab.active {
+    font-weight: $font-weight-bold;
+    border-bottom: $border-width solid $primary;
+}
+
+.modal-footer,
```

Comment:
@Brieuc-brd , these lines will affect the entire frontend ^^ ... Let's scope the entire file, thanks 👍

Reply:
@stefanorigano Indeed 👍 
Since there's no easy way to add a custom class to the `<Dialog/>` (I wanted to avoid using xpath), I wrapped the component into a `div.o_locationSelector`.
What do you think?

Reply:
Ok to me, it suits the current needs.
The goal is indeed to style this backbone layout, the entire feature is not ready for production just yet, we'll see how to improve this specific part during the "real implementation".

Reply:
What do you mean there is no easy way to add a custom class to the `<Dialog/>`? What about `contentClass=` attribute on the XML element?